src/components/tabs/__internal__/tab-title/tab-title.component.js

```diff
@@ -25,7 +25,7 @@ const TabTitle = React.forwardRef(({
   noLeftBorder = false,
   noRightBorder = false,
   customLayout,
-  hasCustomTarget,
+  isInSidebar,
```

Comment:
Just double-checking - can this only be used in Sidebar, or could it potentially be in other components as well?

Reply:
Yeah it is just to apply a style override specific to when it is in the Drawer's sidebar